### ISSUE TITLE
Backports

### ISSUE THREAD

**Issue by Terraphice** [BasiqueEvangelist/Limelight#9]
Figured I'll be the first to ask for back-ports to major modding versions, specifically 1.20.X & 1.19.X.

**Comment by BasiqueEvangelist** [comment]
I actually already made an (unreleased) backport to 1.20.4, and I'll probably make one for 1.20.1.
I dunno if I'll consistently backport new features to those versions, though - having to test on all of them before releasing is something I kind of don't want to do

**Comment by Terraphice** [comment]
I know I would be satisfied with just major changes/releases being backported to only the most recent major modding version, or a stripped down version.

**Comment by Terraphice** [comment]
Just bumping this so you're reminded, since it's been a while since the last release.

And if you do release for 1.20.1, mark it for all the compatible versions above that (namely 1.20.2) as well please. Lots of mod creators forget to do that, when those versions are nearly identical.

**Comment by BasiqueEvangelist** [comment]
Right. I'm kind of, uh, not really feeling like modding in general right now.
I'll try to make a 1.20.1 backport

**Comment by BasiqueEvangelist** [comment]
Okay, wait, it feels like I'll have to also backport the latest owo-lib version to 1.20.1, since that has some features that I use that aren't in the latest owo-lib for 1.20.1. Urgh.

**Comment by Terraphice** [comment]
Don't worry about it. Do it whenever you feel like it, don't push yourself to do something you're not interested in.

**Comment by ShayBox** [comment]
1.21.4 Backport please
Maybe [Stonecutter](https://stonecutter.kikugie.dev) for multi-version support?
